src/components/MessagesList/MessagesGroup/Message/MessagePart/DeckCard.vue

```diff
@@ -21,11 +21,21 @@
 
 <template>
 	<a
-		v-tooltip.bottom="deckLocation"
-		class="external"
+		class="deck-card"
 		:href="link"
+		:aria-label="deckCardAriaLabel"
 		target="_blank">
-		<strong>{{ name }}</strong>
+		<div class="deck-card__lineone">
+			<div class="icon-deck" />
```

Comment:
icon is undefined when deck is disabled. Maybe we can copy and redefine it for that case?

Reply:
Also the icon is black on dark mode 🤷

Reply:
> icon is undefined when deck is disabled. Maybe we can copy and redefine it for that case?

Silly question maybe, but if deck is unavailable what's the point of all this? Shouldn't we display a warning instead of the card?

Reply:
Well ignore that part

Reply:
Dark mode seems healed now 🤔